tests/php81/Php81SerializationTest.php

```diff
@@ -36,4 +37,13 @@ public function testEnum()
             "size": "big"
         }')), $w->toJson());
     }
+
+    public function testBackedEnumArray()
+    {
+        $w = new StatusList();
+        $w->statusList = ['ON', 'OFF', 'ON'];
```

Comment:
This should be a list of `Status` and not a list of string for the test to be valid.